src/main.rs

```diff
@@ -33,6 +33,7 @@ fn user_boot() {
             println!("Could not find '{}'", script);
         } else {
             println!("MFS is not mounted to '/'");
+            println!("Please run 'install' to setup moros");
```

Comment:
```suggestion
            println!("MFS not found, run 'install' to setup the system);
```

Reply:
Happy with this. I also tested and it is as expected. (You missed a " after system :-P)